guide/README.md

```diff
@@ -1,3 +1,5 @@
+---
```

Comment:
These are needed for jekyll to treat like html (and the plugin to handle lack of frontmatter didnt seem to work for README files)